examples/segmentation_demos.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""
+Demo script for anatomical MRI segmentation with SPM.
+
+It demos segmentations on SPM single-subject auditory.
+
+@author: Mehdi RAHIM
+"""
+
+import os
+from pypreprocess.subject_data import SubjectData
+from pypreprocess.datasets import fetch_spm_auditory
+from pypreprocess.nipype_preproc_spm_utils import _do_subject_segment
+from pypreprocess.reporting.check_preprocessing import plot_segmentation
+import matplotlib.pyplot as plt
```

Comment:
I'd do the ``matplotlib`` import just after ``os``, otherwise your plots won't show.